sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -20,10 +20,18 @@ from form documents. It includes the following main functionalities:
 Install the Azure Form Recognizer client library for Python with [pip][pip]:
 
 ```bash
-pip install azure-ai-formrecognizer
+pip install azure-ai-formrecognizer --pre
 ```
 
-> Note: This version of the client library supports the v2.0 version of the Form Recognizer service
+> Note: This version of the client library defaults to the v2.1-preview version of the service
+
+This table shows the relationship between SDK versions and supported API versions of the service
+
+|SDK version|Supported API version of service
```

Comment:
oh cool! Is this something we are doing in other libraries? and languages?

Reply:
So far I think only Python is adding it for clarity since we support multiple API versions in a library and it gets confusing real fast